core/controllers/profile_test.py

```diff
@@ -19,7 +19,9 @@
 from core.domain import exp_services
 from core.domain import user_services
 from core.tests import test_utils
+import datetime
```

Comment:
Oh, sorry, I wasn't clear. "import datetime" should be separated from the other imports, and go before them. We always import python standard libs first -- see 

    https://google.github.io/styleguide/pyguide.html#Imports_formatting

So, the order should be

```
    import datetime

    from core.domain import exp_services
    from core.domain import user_services
    (etc.)
```